base/esg_postgres.py

```diff
@@ -88,19 +77,31 @@ def setup_postgres(force_install=False, default_continue_install="N"):
             return True
         else:
             force_install = True
```

Comment:
force_install is an artifact of the classic installer and it should be fine to be removed as I don't really see much usage for it anymore